search-app/docs/api.md

```diff
@@ -346,6 +346,7 @@ The following fields are specific to the CMR output and most correspond to ECHO1
 | echo:hasVariables (collections only)       | true if there are variables associated with the collection                                                           |
 | echo:hasFormats (collections only)         | true if there are multiple supported formats for any services associated with the collection                         |
 | echo:hasTransforms (collections only)      | true if there are transformations (subset, interpolation or projection) in any of its associated services            |
+| echo:hasSpatialSubsetting (collections only)| true if the setset contains "Spatial" in any of its associated services            |
```

Comment:
"true if any of its associated services support spatial subsetting"

Reply:
Changed.